packages/frontend/app/(dashboard)/course/[cid]/queue/[qid]/page.tsx

```diff
@@ -587,6 +617,15 @@ export default function QueuePage({ params }: QueuePageProps): ReactElement {
             </>
           ) : (
             <>
+              {((queue.type === 'hybrid' && // Show the "Join Zoom" button if staff is ready and student already clicked on the modal
+                studentQuestion?.location === 'Online') ||
+                queue.type === 'online') &&
+                clickedZoomModal &&
+                studentQuestion?.status === OpenQuestionStatus.Helping && (
+                  <JoinZoomButton zoomLink={queue.zoomLink ?? course?.zoomLink}>
+                    Need The Link?
+                  </JoinZoomButton>
```

Comment:
This is really smart that you added this, just a couple small things with it:

```suggestion
                  <JoinZoomButton zoomLink={queue.zoomLink ?? course?.zoomLink}>
                    Join Zoom
                  </JoinZoomButton>
```

Also, the join zoom button is slightly more bold than the other QueueInfoColumn buttons?
![image](https://github.com/user-attachments/assets/b4898731-52ce-4a0c-958b-8ba630d0fddf)